data/changelog.txt

```diff
@@ -566,6 +586,17 @@ Fix assignment ID's with NECC mapgen EOCs.
 Allows Infrared vision to see through smoke
 [Magiclysm] Fix Peaceful Rest hobby
 NPCs can overeat, starve, and dehydrate to death
+[MoM] Update hobby skill numbers
+weakness to water do not disintegrate you anymore
+Eggplants should be treated as veggy for recepies
+Set proper overmap terrain for innawoods cave specials
+Update README.md to remove no longer relevant Bountysource link 
+New fungal enemies should not revive
+Adds nutrients and calories to crafted Sunflower Seeds
+Relocate Russell so that his restocking won't accidentally consume the food storage
+Can't die for real in sky islands (for real this time)
+Camp hunting doesn't bring back human corpses
+Prevent Tae Kwon Do's 'Unhindered' static buff from activating when hands are not empty.
```

Comment:
```suggestion
weakness to water do not disintegrate you anymore
New fungal enemies should not revive
Relocate Russell so that his restocking won't accidentally consume the food storage
Can't die for real in sky islands (for real this time)
Camp hunting doesn't bring back human corpses
```